templates/identifiers/properties.stencil

```diff
@@ -0,0 +1,22 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+
+{% if containers %}
+{% set accessModifier %}{% if param.publicAccess %}public{% else %}internal{% endif %}{% endset %}
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable file_length
+// swiftlint:disable identifier_name
+
+{{accessModifier}} enum AccessibilityIdentifier {
+  {% for container in containers %}
+  {{accessModifier}} enum {{container.name}}: String {
```

Comment:
The fact that `enum` here is declaring `String` rawValue is causing a compiler error, where it says enum with no cases shouldn't declare a rawValue.

Reply:
@Mazyod @IanKeen Would the compiler error go away if we add a conditional statement that checks the size of the array before actually iterating through them?
```
{{accessModifier}} enum AccessibilityIdentifier {
  {% for container in containers %}
    {% if container.accessibility.count > 0 %} 
    {{accessModifier}} enum {{container.name}}: String {
      {% for accessibility in container.accessibility %}
      {{accessModifier}} static let {{accessibility.identifier}} = "{{accessibility.rawValue}}"
      {% endfor %}
    }
    {% endif %}
    {% endfor %}
  }
```
I imagine this would avoid an instance where you get an enum that has no cases.